dali/c_api/c_api.cc

```diff
@@ -57,6 +58,18 @@ int PopCurrBatchSize(batch_size_map_t *batch_size_map, int max_batch_size,
 }
 
 
+dali::DALIExtNoCopyMode GetExternalSourceCopyMode(unsigned int flags) {
+  dali::DALIExtNoCopyMode no_copy_mode = dali::DALIExtNoCopyMode::DEFAULT;
+  DALI_ENFORCE(!((flags & DALI_ext_force_copy) && (flags & DALI_ext_force_no_copy)),
+               "External Source cannot be forced to use copy and no copy at the same time.");
```

Comment:
I feel, that this err msg would be more meaningful, if it would explicitly say, that this is because of `DALI_ext_force_copy` and `DALI_ext_force_no_copy` flags

Reply:
done